src/test/java/net/unicon/lti/controller/lti/OIDCControllerTest.java

```diff
@@ -388,7 +400,7 @@ private void validateOAuthResponse(String response, String clientId, String depl
         assertEquals(OIDC_ID_TOKEN, parameters.get("response_type").get(0));
         assertEquals(OIDC_OPEN_ID, parameters.get("scope").get(0));
         assertTrue(parameters.get("nonce").get(0).length() >= 36);
-        Jws<Claims> finalClaims = Jwts.parser().setSigningKey(kp.getPublic()).parseClaimsJws(parameters.get("state").get(0));
+        /*Jws<Claims> finalClaims = Jwts.parser().setSigningKey(kp.getPublic()).parseClaimsJws(parameters.get("state").get(0));
```

Comment:
Should this be replaced with new checks that the state is getting validated and then this can be deleted?

Reply:
Commented in chat